src/support/azext_support/_params.py

```diff
@@ -102,7 +102,8 @@ def load_tickets_argument(self, _):
                    help='Indicates if this requires a 24x7 response from Azure. Default is false.')
         c.argument('partner_tenant_id', help='Partner tenant id for Admin On Behalf ' +
                    'of (AOBO) scenario. In addition to logging in to the customer tenant, logging in to the partner ' +
-                   'tenant (PT) using "az login -t PT --allow-no-subscriptions" is required.')
+                   'tenant (PT) using "az login -t PT --allow-no-subscriptions" is required.',
+                   deprecate_info=c.deprecate())
```

Comment:
Please run commands manually in command line to check if it's what you want.

Reply:
Verified its what we want